src/NosCore.GameObject/Helper/UpgradeHelper.cs

```diff
@@ -0,0 +1,55 @@
+﻿//  __  _  __    __   ___ __  ___ ___
+// |  \| |/__\ /' _/ / _//__\| _ \ __|
+// | | ' | \/ |`._`.| \_| \/ | v / _|
+// |_|\__|\__/ |___/ \__/\__/|_|_\___|
+// 
+// Copyright (C) 2019 - NosCore
+// 
+// NosCore is a free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or any later version.
+// 
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+// 
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+using System.Collections.Generic;
+using System.Dynamic;
+
+namespace NosCore.GameObject.Helper
+{
+    public sealed class UpgradeHelper
+    {
+        public static UpgradeHelper Instance => _instance ??= new UpgradeHelper();
+        private static UpgradeHelper _instance;
+
+        public byte MaxSumLevel { get; set; }
+
+        public const short SandVNum = 1027;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Change this constant to a 'static' read-only property.](https://app.codacy.com/app/NosCoreIO/NosCore/pullRequest?prid=4591927)